### ISSUE TITLE
Lab 03 - NYT "Strikeouts on the Rise" graph replication

### ISSUE THREAD

**Issue by anaitab** [Watts-College/cpp-526-sum-2024#16]
I would greatly appreciate any hints or help you can give me on these issues and what I am doing wrong. I appreciate any help you can provide.

1- How can I  define an outline for the text box below, and connect it with the labeled data point with a line, as in the NYT graph? 

text(x=1917,y=1,"US enters World War I",col="gray60", cex=0.4, pos=3)

2- I  added the league averages to the graph with the points() function, but the points disappear when I try to add a line to connect the average dots using plot(). How can I overlap the plot to what is already on the graph?

**Comment by JessiMT** [comment]
Hi Ana,

I struggled a lot with this part of the assignment. Here are some posts that I found helpful. The function segment() worked well for me, but make sure to adjust the distance between the segment and the text to avoid any overlap.

https://stackoverflow.com/questions/77239785/creating-a-timeline-plot-represented-as-line-segments
https://www.math.ucla.edu/~anderson/rw1001/library/base/html/segments.html

Jessica
